other/bootstrap_daemon/tox-bootstrapd.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=-/bin/mkdir /var/run/tox-bootstrapd -p
+ExecStartPre=/bin/chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd
+WorkingDirectory=/var/lib/tox-bootstrapd
+ExecStart=/usr/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
```

Comment:
Right. Packages install directly in `/usr/`, but things build from source on your own in `/use/local/`. It's package manager's responsibility to adjust all config files as appropriate for their distribution.